Curtains/Curtains.Domain/Models/OurWorksModel.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Curtains.Domain.Models
+{
+    public class OurWorksModel
+    {
+        public int Id { get; set; }
+        public string Description { get; set; } = null!;
+        public string Image { get; set; } = null!;
```

Comment:
Can we store image in string? 🧐 mb we should use byte[]?

Reply:
Should we store an image in database? Maybe better to store path to it?

Reply:
Good question, seems that Egor Pilyut was responsible for how we will store images. Need to ask him

Reply:
We discussed that and he also thinks that better to store path. I think, we need opinion from Egor Antonovich.